x/posts/internal/types/models/common/post_media_test.go

```diff
@@ -69,35 +86,87 @@ func TestPostMedias_Equals(t *testing.T) {
 					URI:      "uri",
 					MimeType: "text/plain",
 				},
+			},
+			second: common.PostMedias{
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "application/json",
+				},
+			},
+			expEquals: false,
+		},
+		{
+			name: "different length returns false",
+			first: common.PostMedias{
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "text/plain",
+				},
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "application/json",
+				},
+			},
+			second: common.PostMedias{
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "text/plain",
+				},
+			},
+			expEquals: false,
+		},
+		{
+			name: "different tags length returns false",
+			first: common.PostMedias{
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "text/plain",
+					Tags:     []sdk.AccAddress{tag, tag2},
+				},
 				common.PostMedia{
 					URI:      "uri",
 					MimeType: "application/json",
+					Tags:     []sdk.AccAddress{tag},
 				},
 			},
 			second: common.PostMedias{
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "text/plain",
+					Tags:     []sdk.AccAddress{tag},
+				},
 				common.PostMedia{
 					URI:      "uri",
 					MimeType: "application/json",
+					Tags:     []sdk.AccAddress{tag},
 				},
 			},
 			expEquals: false,
 		},
 		{
-			name: "different length returns false",
+			name: "different tags returns false",
 			first: common.PostMedias{
 				common.PostMedia{
 					URI:      "uri",
 					MimeType: "text/plain",
+					Tags:     []sdk.AccAddress{tag2},
 				},
 				common.PostMedia{
 					URI:      "uri",
 					MimeType: "application/json",
+					Tags:     []sdk.AccAddress{tag},
 				},
 			},
 			second: common.PostMedias{
 				common.PostMedia{
 					URI:      "uri",
 					MimeType: "text/plain",
+					Tags:     []sdk.AccAddress{tag},
+				},
+				common.PostMedia{
+					URI:      "uri",
+					MimeType: "application/json",
+					Tags:     []sdk.AccAddress{tag2},
 				},
 			},
 			expEquals: false,
```

Comment:
Can we add a test case to make sure two medias with empty tags return `true` if they are equals?